app/src/main/java/com/example/himanshu/canteen/ChangeItems.java

```diff
@@ -0,0 +1,10 @@
+package com.example.himanshu.canteen;
+
+import android.support.v7.app.AppCompatActivity;
+
+/**
+ * Created by himanshu on 14/2/17.
```

Comment:
Author is automatically recorded in GIT, better leave a comment what this class does.